Library/Homebrew/official_taps.rb

```diff
@@ -15,4 +15,10 @@
   tex
   versions
   x11
-]
+].freeze
+
+OFFICIAL_CMD_TAPS = [
+  ["caskroom", "cask"],
+  ["homebrew", "bundle"],
+  ["homebrew", "services"],
+].freeze
```

Comment:
I think we can just store `homebrew/bundle`. Also if you want to support cmd, maybe:
```ruby
OFFICIAL_CMD_TAPS = {
  "homebrew/bundle" => %w[bundle, ...]
  ...
 }.freeze
```

So for `brew.rb`
```
possible_tap = OFFICIAL_CMD_TAPS.detect do |name, cmds|
  cmds.include? cmd
end.map { |name| Tap.fetch(name) }
```